src/client/app/components/unit/EditUnitModalComponent.tsx

```diff
@@ -71,7 +75,34 @@ export default function EditUnitModalComponent(props: EditUnitModalComponentProp
 		setState({ ...state, [e.target.name]: Number(e.target.value) });
 	};
 
+	/* Confirm Delete Modal */
+	// Separate from state comment to keep everything related to the warning confirmation modal together
+	const [showDeleteConfirmationModal, setShowDeleteConfirmationModal] = useState(false);
+	const deleteConfirmationMessage = translate('unit.delete.unit') + ' [' + props.header + '] ?';
```

Comment:
You are following current usage but I noticed this now. I tried repacing props.header with props.unit.identifier and found it worked fine. Thus, props.header is duplicating the original, other props state. As such, I think it should be removed from the props. I see this also in EditConversionModalComponent so the same change there if you are willing (or I can do).

Reply:
Ah, I was using the conversion code as a template, but there is no identifier in the conversion interface as they use two units and arrows to identify it.  So I have updated the prop in conversion to be called conversionIdentifier and pass it in so it is more telling of what it is.  I got rid of the heading prop for units and just use values.identifier which receives the unit props.  Let me know if this all works